tests/models/test_cpu.py

```diff
@@ -114,7 +114,7 @@ def test_running_test_after_fitting(tmpdir):
     trainer.test()
 
     # test we have good test accuracy
-    tutils.assert_ok_model_acc(trainer)
+    tutils.assert_ok_model_acc(trainer, thr=0.35)
```

Comment:
do we want to define a `DESIRED_THRESHOLD` var in the base file and import that? rather than referencing a float throughout the tests